### ISSUE TITLE
Combining the power of JavaScript and Git

### ISSUE THREAD

**Issue by creationix** [jsfest/sf-cfp#14]
I want to give a talk about all that I've learned through the journey of creating js-git.  This talk can cover topics like:
- How to crowdfund an open project while providing for a family and all that entails.
- Writing large and complex JavaScript code without dying of complexity overload.
- Using git as a database in normal (non dev tool) applications
- Writing JavaScript code in widly varying JavaScript environments
- Much more...

Basically I want to tell the story of where I went, what I learned and what we can use today based on this work as well all the help and support I got from the community.

**Comment by mikeal** [comment]
This is all great but talks are 20 minutes and I know how much you can go over time ;)

Would be good to pick one or two of these points to focus on, that could be done in 20 minutes quite nicely.

**Comment by creationix** [comment]
Right, I have no intentions of going over all those points.  Which part interests you or the other curators?  I can fit content to the time slot given.

**Comment by creationix** [comment]
Also, when did I go over time (I don't remember)?  I generally go under my time when speaking at conferences.

**Comment by mikeal** [comment]
NodeConf 2012 :) But it was a good talk so I didn't stop you.

**Comment by mikeal** [comment]
@creationix also, it's great that you're flexible, having options will help the curators fit it in to the narrative better :)

**Comment by hazeledmands** [comment]
This is my favorite part of your proposal:

> Using git as a database in normal (non dev tool) applications

**Comment by mikeal** [comment]
Another thing just crossed my mind, there's enough here for more than one good talk, I could see putting different talks in to different days if you're up for that.

**Comment by creationix** [comment]
Sure, thing I can do multiple talks if you want.

**Comment by techwraith** [comment]
@creationix Do you want to do this talk for RejectJS on March 8th at Joyent? We'd love to have you :)

**Comment by creationix** [comment]
@Techwraith Sure, but I would need help getting there.  I can't afford a plane ticket or hotel right now.

**Comment by techwraith** [comment]
Unfortunately we're running this on a tight budget, we can't help speakers
get to this event. Thanks for submitting this though! If you know anyone
else who would be able to give this talk, let me know.

**Comment by creationix** [comment]
@Techwraith I understand.  Thanks for thinking of me.  I'm afraid most the content is stuck in my head at the moment.  I'm not sure how someone else could give the talk for me. (also most the js-git collaborators are outside the bay area too)
